resource_packs/files/utility/brewing_guide/uibt/brewing_icons.ui

```diff
@@ -254,6 +254,14 @@
     "$item_texture": "textures/items/turtle_helmet",
     "$hover_text": "item.turtle_helmet.name"
   },
+  "breeze_rod@multi_item_panel": {
+    "$item_texture": "textures/items/breeze_rod",
+    "$hover_text": "item.breeze_rod.name"
+  },
+  "stone@multi_item_panel": {
+    "$item_texture": "textures/blocks/stone",
```

Comment:
change to a 3d item, can take a high res screenshot and edit the stone out

Reply:
Done, and the textures are 114x126 and i used blockbench to render it
![Screenshot 2024-06-28 223932](https://github.com/BedrockTweaks/Files/assets/85175982/f26fbde0-5fed-4f81-b681-8eeca8baa96f)